src/visualizr/anitalker/networks/encoder.py

```
@@ -1,447 +1,447 @@
+            self.convs.append(ResBlock(in_channel, out_channel))
+            in_channel = out_channel
+
+        self.convs.append(EqualConv2d(in_channel, self.w_dim, 4, bias=False))
+
+        self.fusion_type = fusion_type
+
+        if self.fusion_type != "weighted_sum":
+            raise ValueError(
+                f"Unsupported `fusion_type`: {self.fusion_type}. "
+                "Expected 'weighted_sum'."
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)